ledger/eval/eval_test.go

```diff
@@ -1401,178 +1445,3 @@ func TestExpiredAccountGeneration(t *testing.T) {
 	require.Equal(t, crypto.VRFVerifier{}, recvAcct.SelectionID)
 	require.Equal(t, merklesignature.Verifier{}.Commitment, recvAcct.StateProofID)
 }
-
-func TestEval_EndOfBlockStake(t *testing.T) {
```

Comment:
It would have been nice to keep the test here

Reply:
IMO this test too bound to vetted blocks on mainnet. We can resurrect it if decide to re-vet.